.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
Tip: You can add this to your local `.git/info/excludes` file - that way you can ignore stuff locally without having to update the (shared) gitignore file in the repo. Anyway, if you want to keep this fine, but please also include these other common excludes in the file:

```gitignore
*~
*.swp
*.sublime-*
.idea/
.vscode/
```

Reply:
Ok, I'll add them. I think it makes life easier to exclude the "usual" things...